### PR TITLE
Update ui multi player

### PR DESCRIPTION
**Pull request by bryanvanwijk**
I added control explanation to multi and single player.
Add the different spaceship for player 2 to the game and decreased the amount of heart powerups.

### CODE REVIEW

File: src/main/resources/spaceinvaders/group_22/ui/MultiPlayerGameUI.fxml [hunk 1]
```diff
@@ -112,12 +152,22 @@
                      </image>
                   </ImageView>
                   <Canvas fx:id="canvas" height="720.0" layoutX="140.0" onKeyPressed="#handleKeyPressed" onKeyReleased="#handleKeyReleased" styleClass="canvas" width="1000.0" />
-                  <Label fx:id="scoreLabel" layoutX="150.0" prefHeight="50.0" prefWidth="200.0" textFill="WHITE">
+                  <Label text="Player 1:" layoutX="150.0" prefHeight="50.0" prefWidth="200.0" textFill="#8d90ff">
```
Comment (jochem725) [inline src/main/resources/spaceinvaders/group_22/ui/MultiPlayerGameUI.fxml]: I think the ':' can be left out here, it looks smoother that way.

**Comment by jochem725** [comment]
PMD reports a few useless parenthesis, maybe these can be removed.
After that has been changed, I think this can be merged.

**Comment by bryanvanwijk** [comment]
I fixed the points you mentioned..

**Comment by jochem725** [comment]
Looks great! I'll agree with merging.

**Comment by doriandekoning** [comment]
Looks very nice and makes the game clearer. I agree with merging too.

**Pull request merged**